brave-spark-framework-filter/src/main/java/com/github/kristofa/brave/spark/SparkHttpServerResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.http.HttpResponse;
+import spark.Response;
+
+/**
+ * Created by 00013708 on 2017/3/10.
+ */
+public class SparkHttpServerResponse implements HttpResponse {

```

Comment:
types like these don't need to be public (ps remove the automated javadoc please)